main.cpp

```diff
@@ -16,11 +16,16 @@
  * License along with this library.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-#include "simplecpp.h"
-
+#include <cstdlib>
+#include <cstring>
 #include <fstream>
 #include <iostream>
-#include <cstring>
+#include <map>
+#include <string>
+#include <utility>
+#include <vector>
+
+#include "simplecpp.h"
```

Comment:
It's an advantage to put local headers before the system includes.

the local header might use types in the system includes and then it's good to enforce that there are corresponding includes in the header.

Reply:
I accidentally did it the wrong around as `simplecpp.h` is a "system" header in the Cppcheck code base - which this is not...

Reply:
Done.